build/patches/Bromite-AdBlockUpdaterService.patch

```diff
@@ -405,6 +604,39 @@ diff --git a/chrome/app/generated_resources.grd b/chrome/app/generated_resources
 +      <message name="IDS_OPTIONS_ADBLOCK_SUMMARY" desc="The title of the Ad Blocking summary on Android" formatter_data="android_java">
 +        Configure Ad Blocking and filters URL
 +      </message>
++      <message name="IDS_OPTIONS_ADBLOCK_CURRENT_INDEX_TITLE" desc="The title of the Ad Blocking current index version on Android" formatter_data="android_java">
++        Current Indexed Filter Version
++      </message>
++      <message name="IDS_ADBLOCK_LAST_CHECKED_TEXT" desc="The title of the Ad Blocking last checked datetime" formatter_data="android_java">
++        Last Checked:
++      </message>
++      <message name="IDS_OPTIONS_ADBLOCK_FREQUENCY_TITLE" desc="The title of the Ad Blocking frequency to check update" formatter_data="android_java">
++        Check every
++      </message>
++      <message name="IDS_OPTIONS_ADBLOCK_FREQUENCY_DAY" desc="Ad Blocking frequency single day" formatter_data="android_java">
++        Day
++      </message>
++      <message name="IDS_OPTIONS_ADBLOCK_FREQUENCY_DAYS" desc="Ad Blocking frequency multiple day" formatter_data="android_java">
++        Days
++      </message>
++      <message name="IDS_OPTIONS_ADBLOCK_STARTCHECK_LABEL" desc="The title of the Ad Blocking button to check update" formatter_data="android_java">
++        Check Now
++      </message>
++      <message name="IDS_OPTIONS_ADBLOCK_EVENT_1" desc="The text for ADBLOCK_CHECKING_FOR_UPDATES" formatter_data="android_java">
++        Adblock filter: checking for update
++      </message>
++      <message name="IDS_OPTIONS_ADBLOCK_EVENT_3" desc="The text for ADBLOCK_UPDATE_DOWNLOADING" formatter_data="android_java">
++        Adblock filter: downloading update...
```

Comment:
```suggestion
+        Downloading new filters...
```

Reply:
fixed in https://github.com/bromite/bromite/pull/905/commits/80c163754a3afa025bd7004aec20d43df53a4c12